pynitrokey/fido2/client.py

```diff
@@ -33,25 +33,33 @@
 
 
 class NKFido2Client:
-    def __init__(self,):
+    def __init__(
+        self,
+    ):
         self.origin = "https://example.org"
         self.host = "example.org"
         self.user_id = b"they"
         self.exchange = self.exchange_hid
         self.do_reboot = True
 
-    def use_u2f(self,):
+    def use_u2f(
+        self,
+    ):
```

Comment:
Decreases readability (this one and other similar)

Reply:
Looks like this is caused by the trailing comma.  Any reason to have that there?

Reply:
Possibly. No idea about the reason - never seen that before.

Reply:
I think this is the rule that is applied here:

> collection literals with trailing commas will be kept one-per-line instead of being folded onto a single line if they fit

https://github.com/psf/black/pull/826/commits/ff41e949da7a6508cf5ff134b1808483578aceab

Reply:
Here is the relevant section from the style guide: [The magic trailing comma](https://black.readthedocs.io/en/latest/the_black_code_style/current_style.html#the-magic-trailing-comma).